program/rust/src/tests/test_upd_aggregate.rs

```diff
@@ -78,13 +77,38 @@ fn test_upd_aggregate() {
     price_account.is_signer = false;
     PriceAccount::initialize(&price_account, PC_VERSION).unwrap();
 
+    // test same slot aggregation, aggregate price and conf should be updated
+    {
+        let mut price_data = load_checked::<PriceAccount>(&price_account, PC_VERSION).unwrap();
+        price_data.num_ = 1;
+        price_data.last_slot_ = 1000;
+        price_data.agg_.pub_slot_ = 1000;
+        price_data.comp_[0].latest_ = p1;
+    }
+    unsafe {
+        assert!(c_upd_aggregate(
+            price_account.try_borrow_mut_data().unwrap().as_mut_ptr(),
+            1000,
+            1,
+        ));
```

Comment:
Did you mean to add some asserts after this?

Reply:
yes 🤦‍♂️